src/main/java/core/basesyntax/Application.java

```diff
@@ -2,6 +2,10 @@
 
 public class Application {
     public static void main(String[] args) {
-        // create three balls using class Lottery and print information about them in console
+        for (int a = 0; a < 3; a++) {
+            Ball ball = new Ball();
+            ball = new Lottery().getRandomBall();
+            System.out.println(ball.toString());
```

Comment:
1. create a lottery before loop and reuse it. 
2. Don't create ball instance manually at all, use what methods returns to you
3. toString call is redundant